data/kahet/kahet missions.txt

```diff
@@ -738,3 +738,1160 @@ event "pacified ka'sei"
+
+
+conversation "end of Patir 2 - on complete"
+	`A large crowd has quickly gathered around your landing pad, eagerly waiting your arrival. After all this time between the stars, setting foot on a planet with an actual atmosphere is incredibly refreshing. Ignoring the cold weather for a moment, you take the biggest breath you can to make up for the past weeks of recycled atmosphere.`
+	`	Before you've even reached the end of the ramp, the people on the ground are already signing questions to you and the rest of the team about your journey. You do your best to reply, but in the overall mix of confusion and excitement you aren't sure anyone understood what you just said.`
+	`	Between the chaos, you hear a familiar voice singing your name: a few meters on your left, you notice Chilia with his arm raised and a large grin greeting you back. Moving across the crowd not without some difficulty, you follow him through a narrow door inside a cramped room right next to the pad.`
+	`	"I prefer this spaceport when it is more hushed," he remarks with a sigh, "but after the past weeks some frenzy is understandable. We will have to make do with this warehouse for now." He invites you to take a seat. "So, <first>, would you tell me about your journey?"`
+	`	By the time you're done, more than an hour must have passed. Chilia paces aimlessly around the room, reflecting on your recounting of the events. As silence falls, you notice that calm seems to have returned on the pad as well.`
+	choice
+		`	(Wait for him to speak.)`
+			goto wait
+		`	"Do you know how we came back to this galaxy?"`
+		`	"What happened while we were gone?"`
+	`	"Oh. I thought you had been told already." He sits down in front of you. "It was us that brought Patir back here. After your departure, it did not take long for our long-range sensors to pick up that the black hole had disappeared, along with you and the team. Our data was scarce, but we knew one, very important thing: if Patir had appeared here once, there must have been a way to do that again.`
+		goto explain
+	label wait
+	`	"If your description is accurate, it means Patir is not another mouth to a pocket universe, but instead a transporter to another galaxy. Or universe, for all we know. In retrospect, there were enough reasons to doubt that first theory, but we were far too busy trying to save you to look into it seriously." Seeing your confused expression, he continues. "<first>, it was us who brought Patir back here.`
+	label explain
+	`	"We knew Postverta was involved with Patir's initial arrival, and together with the team there, we discovered that the energy burst that had been released when Ssil Vida first appeared had no correlation with the systems controlling its pocket universe. So, its activation or deactivation would have no effect on Patir, and we had to start looking somewhere else.`
```

Comment:
Sorry if I'm misinterpreting something, but if the energy burst that was released when Ssil Vida appeared did not affect Patir or other systems, then why did the Remnant try to replicate that energy burst on Ssil Vida in an attempt to bring back Patir?

Reply:
From Patir 1:
`"I only have theories for now, none of which can be confirmed. People more knowledgeable than me in the subject are examining the phenomenon right now. But look at this," she says, opening a star map on a holographic projector. "This white dot is Postverta, and we are here. When Ssil Vida first appeared, it released a strong burst of ionizing radiation, and when we were able to calculate its velocity, we found it was going much faster than the speed of light. It is gradually slowing down, so it will not go much further, but exactly the same day that the energy burst was supposed to arrive here, Patir appeared! It would be an incredible coincidence if the two events were not connected.`

Reply:
But doesn't this contradict line 1872? What do you mean by 'no correlation with the systems controlling its pocket universe' in line 1872? Sorry if I'm being a bit dense here.

Reply:
No worries. The meaning there was that the initial burst that revealed the system to you was the one that made Patir appear, but when the Remnant later learned how to move Ssil Vida back and forth between dimensions, that function had no relation with Patir itself; that is, whether the Remnant or the player activate or deactivate Ssil Vida has no influence on Patir's appearance or disappearance, and it took some time and effort on the part of the Remnant for them to get Ssil Vida to emit the right signal again

Reply:
Ah I see. Sorry, it's been a while since I've played through Remnant content. Maybe this would make it a bit clearer?

```suggestion
	`	"We knew Postverta was involved with Patir's initial arrival, and together with the team there, we discovered that the energy burst that had been released when Ssil Vida first appeared had no relation to the systems controlling its pocket universe. So, its activation or deactivation would have no effect on Patir, and we had to start looking somewhere else.`
```